streamer/parser/a11y-metadata-parsing.md

```diff
@@ -74,91 +81,111 @@ When it is a multiple:
 
 ## CertifierCredential
 
-`certifierCredential` is a key whose value is an array of strings.
+`certifierCredential` is a key whose value is a string
 
 ### EPUB 2.x
 
-The array is created from the `meta` elements whose `name` attribute has the value `a11y:certifierCredential`. The value of their `content` attribute is pushed to the array.
+The string is the value of the first `meta` element whose `name` attribute has the value `a11y:certifierCredential`.
 
 ### EPUB 3.X
 
-The array is created from: 
+- Check if the `certifiedBy` element that has been picked is refined by any `meta` whose `property` attribute has the value `certifierCredential`.
+- If not, use the first `meta` element whose `property` attribute has the value `certifierCredential`.
 
-- the `meta` elements whose `property` attribute has the value `a11y:certifierCredential`. Their value (i.e. child textual content) is pushed to the array;
-- the `link` elements whose `property` attribute has the value `a11y:certifierCredential`. The value of their `href` attribute is pushed to the array – the value is expected to be a URL.
+The string is the textual value of this `meta` element. 
 
 ## CertifierReport
 
-`certifierReport` is a key whose value is an array of strings.
+`certifierReport` is a key whose value is a string.
 
 ### EPUB 2.x
 
-The array is created from the `meta` elements whose `name` attribute has the value `a11y:certifierReport`. The value of their `content` attribute is pushed to the array.
+The string is the value of the first `meta` element whose `name` attribute has the value `a11y:certifierReport`.
 
 ### EPUB 3.X
 
-The array is created from the `link` elements whose `property` attribute has the value `a11y:certifierReport`. The value of their `href` attribute is pushed to the array – the value is expected to be a URL.
+- Check if the `certifiedBy` element that has been picked is refined by any `link` whose `rel` attribute has the value `a11y:certifierReport`.
+- If `certifiedBy` is not refined by any suitable `link`, use the first `link` whose `rel` attribute has the value `a11y:certifierReport`.
+
+The string is the textual value of this `link` – the value is expected to be a URL.
 
 ## ConformsTo
 
 `conformsTo` is a key whose value is an array of strings.
 
-These strings must be valid URLs. 
+Values are likely to point to some version of the EPUB accessibility specification and some WCAG profile.
+Try to canonicalize the values to well-known URLs.
 
-Although this URL can be arbitrary, it is likely one of the following:
+Replace any of the following strings with "http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-a":
+- "EPUB Accessibility 1.1 - WCAG 2.0 Level A"
+- "http://idpf.org/epub/a11y/accessibility-20170105.html#wcag-a"
+- "http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-a"
+- "http://wwwidpf.org/epub/a11y/accessibility-20170105.html#wcag-a"
```

Comment:
I guess this instead?

```suggestion
- "https://idpf.org/epub/a11y/accessibility-20170105.html#wcag-a"
```

Reply:
Thanks. Stupid copy-paste from old code already fixed.